README.md

```diff
@@ -848,6 +848,25 @@ const Screen = () => (
 )
 ```
 
+### 7.5 Styled Component Naming Convention
```

Comment:
Please, add this topic in section table of content

Reply:
@mauriciomutte, I forgot that, sorry.
Done: 6b8fbf21fd66fdc8ea6272a8466b7f2d36016c6b